tests/unit/test_tags.py

```diff
@@ -62,8 +62,8 @@ def test_sort_tags() -> None:
 
 ##############################################################################
 @mark.parametrize("string", ("", "1"))
-def test_tag_langth(string: str) -> None:
-    """We should be able to get the len of a tag."""
```

Comment:
Actually, no, `len` is correct here; it's testing if we can get the `len` of the tag -- as in it should be possible to call the function `len` on the tag.